docs/reference/commandline/swarm_join_token.md

```diff
@@ -65,6 +66,12 @@ SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-b30ljddcqhef9b9v4rs7
 ### `--rotate`
```

Comment:
Before this add:

`Node that tokens allow new nodes to join the swarm and should be kept secret. You should be particularly careful with manager tokens, since they allow nodes to join the swarm as managers, giving them the potential to disrupt the operation of your swarm.`